schema.sql

```diff
@@ -3,7 +3,7 @@ CREATE TABLE books(
   id SERIAL PRIMARY KEY,
   title VARCHAR(255),
   author VARCHAR(255),
-  image_url VARCHAR(255),
+  image_url TEXT,
```

Comment:
I'm changing it back to VARCHAR after reading your comment and Stack Overflow. The original problem was that all of our image urls were so very long, much longer than 255 characters.